src/main/java/com/xjeffrose/chicago/DBManager.java

```diff
@@ -145,7 +155,9 @@ boolean write(byte[] colFam, byte[] key, byte[] value) {
       log.error("Tried to write a null value");
       return false;
     } else if (!colFamilyExists(colFam)) {
-      createColumnFamily(colFam);
+      synchronized (columnFamilies) {
```

Comment:
Actually this issue occurred with the concuurent map as well. 2 clients tried to create a colFam and 1 failed. This is to avoid that.